src/games/strategy/triplea/ui/menubar/ExportMenu.java

```diff
@@ -128,7 +133,7 @@ private void exportXMLFile() {
 
   private void addSaveScreenshot(final JMenu parentMenu) {
     final AbstractAction abstractAction = SwingAction.of("Export Map Snapshot", e -> {
-
+      // get current history node. if we are in history view, get the selected node.
```

Comment:
This comment was originally in `saveScreenshot()` and appeared to have been orphaned there when the following code was moved to this location at some point in the past.  I moved it simply to avoid confusion in `saveScreenshot()`.